javascript/first-js-oop-project.md

```diff
@@ -0,0 +1,156 @@
+# Mini Project: Object Oriented Programming in JS
+
+### Projected Time
+
+Total: 65 min - 80min
+- Independent Practice: 15 min
+- Materials: 25 min
+- Lesson: 30 min
+
+### Prerequisites
+
+- [Java script lesson 7](../javascript/javascript-7-oop.md)
+
+
+### Objectives
+
+- To use the new JS object-oriented programming skills.
+- To create a few event objects and push them to an array.
+- Use jQuery to display the list of events to an html page.
+
+### Specific Things To Teach
+
+- Creating event array.
+- Iterating through the array.
+- Displaying array items in HTML page.
+
+### Materials
+
+- [Documentation on each statement of jQuery](http://api.jquery.com/jquery.each/)
+- [Arrays in javaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)
+- [.html function of jQuery](http://api.jquery.com/html/)
+
+### Lesson
+
+- The below example is the just a part of a mini project that will help you to learn basic concept about arrays in javaScript and iterating through the arrays. The below code will allow you to learn about displaying the array objects in the web pages.
+- Once you are done with the code, run the index.html file in your browser to check the result.
+- Keep on refreshing the browser to keep track of the changes you make everytime.
+- Follow the steps described below:
+
+1. Create a js file `event.js`.
+2. Create an empty Event array.
+```javascript
+var EventArray = new Array();
+
+```
+3. Create an Event object and push the object to the array. In order to push objects in the array, do the following.
+```javascript
+var EventObj =
```

Comment:
only capitalize classes. 
Please take this out, and use the class to create each event object, and push each event object into a regular array.